app/assets/javascripts/listings/components/listing/process-section.html.slim

```diff
@@ -24,8 +24,7 @@ accordion-heading
             | {{::session.Date | dateSuffix}}
           span.content-group_time
             | {{::session.Start_Time}}
-        p.content-group_address.c-steel.no-margin.skiptranslate ng-if="::session.Venue"
-          | {{::session.Venue}}
+        p.content-group_address.c-steel.no-margin.skiptranslate ng-if="::session.Venue" ng-bind-html="::session.Venue | stripMostTags | nl2br"
```

Comment:
I think this is for info session venues vs open house venues. I'll ask if we want to update those too, but for now they are not being updated in salesforce

Reply:
asked in slack here: https://sfinnovation.slack.com/archives/CPZHGET3R/p1596818833032800

Reply:
Decision was to go ahead and support rich text in this as well as we might want links here in the future.